.eslintrc

```diff
@@ -74,7 +74,7 @@
         "except": [ "./migration/migrations" ]
       }, {
         "target": "./assets/src/edit-story",
-        "from": "./assets/src/edit-story/utils/useWhyDidYouUpdate.js"
+        "from": "./packages/story-editor/src/utils/useWhyDidYouUpdate.js"
```

Comment:
There are two more references to `assets/src/edit-story` right above that need updating.

Reply:
https://github.com/google/web-stories-wp/pull/8573/commits/0cb83f0fc0bdd7d17603ee88055e76dc0d76c824